docs/Belegverarbeitung/Belegverarbeitung im RZL Board/Grundeinstellungen.md

```diff
@@ -0,0 +1,160 @@
+*Gebucht* in welcher Reihenfolge angezeigt werden sollen.  
+  
+![](img/image59.png)
+
+-   **Dateiname:** Name, unter welcher die Datei gespeichert ist
+
+-   **Klient:** Nummer und Name des Klienten
+
+-   **Belegkreis:** zB ER, AR, KA, …
+
+-   **Belegart:** Eingangsbeleg, Ausgangsbeleg …
+
+-   **Belegnummer:** fortlaufende Nummer
+
+-   **Fremdbelegnummer:** Rechnungsnummer des Beleges
+
+-   **Belegdatum:** Datum des Beleges
+
+-   **Gesamtbetrag:** Bruttobetrag
+
+-   **Buchungssatz:** Soll/Haben
+
+-   **Gebucht von:** Name des Erfassers
+
+-   **Gebucht am:** Datum der Erfassung
+
+-   **Buchungstext:** erfasster Text beim Buchen
+
+-   **Kommentar:** Vermerk der beim Beleg erfasst wird
+
+-   **Importdatum:** wann der Import stattgefunden hat
+
+-   **Importiert von:** von dem der Import durchgeführt wurde
+
+-   **Belegeingangsdatum:** Buchungsdatum
+
+-   **Bezahlt am:** Datume der durchgeführten Zahlung
+
+-   **Quelle:** ect. …
+
+Im Feld links *ausgewählte Felder für Stempel* kann die Reihenfolge der
+ausgewählten Felder geändert werden.
+
+Setzt man den Haken *Vorschau mit Beispieldaten befüllen* wird ein
+Musterstempel mit Beispieldaten befüllt.
+
+![](img/image60.png)
+![](img/image61.png)
+![](img/image62.png)
+
+Rufen Sie einen bereits gebuchten in der FIBU auf, können Sie in der
+Menüleiste entscheiden, ob der Buchungsstempel sichtbar angezeigt wird.
+
+![](img/image63.png)
+
+Beim Export eines Beleges als PDF wird der Stempel und der Kommentar
+nicht exportiert und angezeigt.
+
+**Grundeinstellungen**
+
+![](img/image64.png)
+
+-   **Klientenportal-Belege:** Mit dieser Option wird beim Öffnen eines
+    Klienten in der Menüleiste angezeigt, wie viele Belege vom
+    Klientenportal noch abzuholen sind.
+
+-   **Finmatics Belege:** Mit dieser Option wird beim Öffnen eines
+    Klienten in der Menüleiste angezeigt, wie viele Belege von Finmatics
+    noch abzuholen sind.
+
+-   **Name/Anschrift mit UID-Nummer über FinanzOnline auslesen:** Mit
+    dieser Option wird bei einer Belegdatenerkennung der Name und die
+    Anschrift des Rechnungsaussteller (bei Eingangsbelegen) bzw. des
+    Rechnungsempfängers (bei Ausgangsbelegen) über die UID-Nummer bei
+    FinanzOnline ausgelesen. Somit haben Sie später in der Liste der
+    *ungebuchten Belege* eine bessere und schnellere Übersicht über die
+    Belege.  
+    Die einmal vorgekommenen UID-Nummern werden für eine gewisse Zeit
+    zwischengespeichert, damit bei Lieferanten, die häufig vorkommen,
+    nicht jedes Mal bei FinanzOnline abgefragt werden muss.
+
+-   **GLN der Lieferanschrift als Kundennummer des Rechnungsempfängers
+    auslesen:** dies ist eine spezielle Funktion beim Import von
+    ebInterface XML-Rechnungen. Dies kann genutzt werden, um später in
+    der FIBU auf Basis dieser GLN-Nummer Buchungsvorschläge zu
+    generieren.
+
+# Zusatzmodul Belegfreigabe
+
+Siehe Kurzanleitung *RZL Modul Belegfreigabe* unter *Hilfe /

```

Comment:
vielleicht tatsächlichen externen Link hier einfügen -https://rzlsoftware.at/fileadmin/user_upload/PDF_Kurzanleitungen/BELEG/BELEGE_Belegfreigabe.pdf
Bzw. irgendwann werden wir die Kurzanleitung so wie in Fibu Next auch vil direkt im dem Handbuch ergänzen - aber das musst jetzt nicht du machen

Reply:
Eigentlich ist im Handbuch das Zusatzmodul Belegfreigabe nicht unter Grundeinstellungen sondern ein eigener Punkt
Magst du bitte auch eine eigene Seite für das Zusatzmodul Belegfreigabe einfügen. Danke